test/blackbox/test_case_optimal_pmtu_01/.gitignore

```diff
@@ -0,0 +1,9 @@
+.deps
+.libs
+Makefile.in
+Makefile
+*.o
+node_sim_nut
+node_sim_peer
+node_sim_relay
+test/
```

Comment:
We shouldn't need these .gitignore files, since `test/blackbox/.gitignore` already ignores all those files, and ignores are applied inside subdirectories as well.